.dev/runTests.R

```diff
@@ -0,0 +1,18 @@
+#
+# you can run this in a R container...
+# see ./test-container/README
+#
+
+# If testing plot.data, load plot.data environment (handle time zone, for example)
+library(dotenv)
+load_dot_env(file=".dev/.env")
+
+# Load devtools (allows testing)
+library(devtools)
+
+# Load local library.
+devtools::load_all("../")
```

Comment:
this file is just generally a bit odd to me.. its presence in veupathUtils suggests it should run veupathUtils tests and use a veupathUtils .env file. but that doesnt seem to be the case. is this meant to be general purpose/ used in multiple repos? if so that would mean including a .env file in our R packages would become part of our SOP for making them if its not already.. and i also wonder whats the advantage of doing this over just starting an interactive session, loading the lib you want to test and calling test() directly?

Reply:
in some ways i see this as possibly belonging in Rserve or technical docs somewhere. I like Rserve because that's the production side of our testing environment. Adding it all to technical docs is a bit of a bummer because it's not tracked so nicely, but at least keeps the repos clean.

Alternatively we could do an environment for each repo, only have stuff related to that repo in there, and then in technical docs have something about running them all together?

@bobular which would be more helpful for you? To have each repo have its own little setup or to have one place where we have all of the repos together? Or a bit of both?